schemas/groups/propulsion.json

```diff
@@ -213,6 +213,21 @@
               "description": "Average fuel rate of consumption",
               "$ref": "../definitions.json#/definitions/numberValue",
               "units": "m3/s"
+            },
+            "vacuum": {
+              "description": "Vacuum measured at the fuel filter - an indication of the state of the fuel filter. The higher the vacuum, the more clogged the filter.",
+              "$ref": "../definitions.json#/definitions/numberValue",
+              "units": "Pa"
+            },
+            "current": {
```

Comment:
I have trouble having _current_ and _voltage_ under `fuel`. Maybe they should go one level up in the hierachy?

Furthermore we generate the release notes from PRs and it would make much more sense if there were separate entries for vacuum and support for electrical engines.

Reply:
Yeah, not sure what I was thinking - maybe amps and volts at the fuel pump? But to be honest, I did this PR so long ago, I don't even have the git files that this change came from. And the `vacuum` item isn't really all that important - I just did it for my own boat, honestly. :-) Feel free to close this PR if you don't want to mess with making the change yourself. Sorry, @tkurki !